tests/chainermn_tests/functions_tests/test_point_to_point_communication.py

```diff
@@ -44,8 +46,9 @@ def create_communicator(gpu, param):
 
 
 def create_x(gpu, param, communicator):
+    io_dtype = chainer.get_dtype(dtype=param.dtype, map_mixed16=numpy.float16)
```

Comment:
What is intended by explicitly setting `map_mixed16` here?

Reply:
The same as the other PR